src/frontend/web_application/src/scenes/MessageList/components/Message/style.scss

```diff
@@ -1,160 +1,109 @@
 @import '../../../../styles/vendor/bootstrap_foundation-sites';
-@import '../../../../styles/object/o-avatar';
 @import '../../../../styles/object/o-reset-html-message';
 
+$message_content_width: 45rem;
+
 .m-message {
   @include flex-grid-row($size: expand);
   margin-bottom: $co-grid__gutter--small;
 
-  &__avatar-col {
-    @include flex-grid-column(shrink);
-    display: none;
+  &__info {
+    @include flex-grid-column(12);
     padding-top: $co-component__spacing;
+    padding-bottom: $co-component__spacing;
+    background-color: $co-color__fg__back--low;
+    font-size: $co-font__size--xsmall;
+  }
+
+  &__from,
+  &__pi {
+    @include flex-grid-row;
+    align-items: center;
   }
 
-  &__avatar { @include m-avatar--size(medium); }
+  &__info-label {
+    @include flex-grid-size(shrink);
+    margin-right: $co-component__spacing;
+  }
+
+  &__info-author {
+    @include flex-grid-size;
+    color: $co-color__fg__text--low;
+    font-weight: 600;
+  }
 
   &__container {
-    @include flex-grid-column;
-    padding-right: 0;
-    padding-left: 0;
+    @include flex-grid-size(12);
+    padding: 0 $co-grid__gutter--small $co-grid__gutter--small;
     background: $co-color__fg__back;
   }
 
   &__top-bar {
-    @include flex-grid-row($size: expand);
+    @include flex-grid-row;
+    flex-wrap: nowrap;
     align-items: center;
-    border-bottom: 1px solid $co-color__fg__back--lower;
-    background: $co-color__fg__back;
+    padding: $co-component__spacing 0;
+    border-bottom: 1px solid $co-color__fg__border--higher;
+    color: $co-color__fg__text--high;
     font-size: $co-font__size--small;
-
-    &--is-unread {
-      background: $co-color__fg__back;
-    }
   }
 
-  &__with-attachements { @include flex-grid-column(shrink); }
   &__type { @include flex-grid-size(shrink); }
   &__type-icon { color: $co-color__fg__text; }
   &__type-label { display: none; }
 
-  &__pi { @include flex-grid-size(shrink); }
-
-  &__author {
-    @include flex-grid-column;
-    color: $co-color__fg__text--high;
-    font-weight: 600;
-  }
-
-  &__date {
-    @include flex-grid-column(shrink);
-    margin-left: auto;
-  }
-
-  &__actions-switcher {
+  &__participants {
     @include flex-grid-size(shrink);
-    border-left: 1px solid $co-color__fg__back--lower;
+    margin-left: auto;
   }
 
-  &__subject {
-    padding: $co-component__spacing;
-    border-bottom: 1px solid $co-color__fg__back--lower;
-    color: $co-color__fg__text--lower;
+  &__participant {
+    &::after { content: ', '}
```

Comment:
I'm not sure it a good idea,
this gives a responsibility of the content to the stylesheets which is initially the job of react and may be accessibility softwares might not interpret it (I have no idea).

Reply:
this what I meant : https://css-tricks.com/css-content/

1/ "content not design"
2/ "Browser support / Accessibility"